index.js

```diff
@@ -181,7 +181,14 @@ module.exports = {
     "react/jsx-no-undef": 2,
     "react/jsx-pascal-case": 2,
     "react/jsx-sort-prop-types": 0,
-    "react/jsx-sort-props": 0,
+    "react/jsx-sort-props": [2, {
+        "noSortAlphabetically": true,
+        "shorthandFirst": true
+      }
+    ],
+    "react/jsx-no-duplicate-props": [2, {
```

Comment:
please check if this rule is supported by lowest specified `peerDependancies`, if yes, LGTM

Reply:
@gh3r mówi się LGBT, nie LGTM

Reply:
Confirmed